src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/app';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
ReactDOM.render is no longer supported in React 18. Use createRoot instead
```suggestion
const root = ReactDOM.createRoot(document.getElementById('root'))
root.render(
    <App />
)